ansible/build-vars.yml

```diff
@@ -0,0 +1,9 @@
+nodejs_app_name: universal
+nodejs_app_install_dir: /opt/{{ nodejs_app_name }}
```

Comment:
I think that might for the real app (prefs, flow manager, etc).

    $ ls
    Dockerfile     gpii          index.js      package.json  tests
    documentation  gpii.js       LICENSE.txt   README.md
    examples       Gruntfile.js  node_modules  testData

    $ tree -L 2 -P node_modules .
    .
    |-- documentation
    |-- examples
    |   |-- deviceReporter
    |   |-- easit-oauth-integration
    |   |-- gpii-oauth2-resource-server
    |   |-- gpii-oauth2-sample-client
    |   |-- gpii-oauth2-sample-client-passport
    |   |-- gpii-oauth2-sample-data
    |   |-- gpii-oauth2-single-process-auth-server
    |   `-- preferences
    |-- gpii
    |   |-- configs
    |   `-- node_modules
    |-- node_modules
    |   |-- body-parser
    |   |-- connect-ensure-login
    |   |-- dedupe-infusion
    |   |-- express
    |   |-- express-handlebars
    |   |-- express-session
    |   |-- grunt
    |   |-- grunt-contrib-jshint
    |   |-- grunt-gpii
    |   |-- grunt-jsonlint
    |   |-- grunt-shell
    |   |-- infusion
    |   |-- ini
    |   |-- kettle
    |   |-- node-jqunit
    |   |-- node-uuid
    |   |-- oauth2orize
    |   |-- passport
    |   |-- passport-local
    |   |-- passport-oauth2-client-password
    |   |-- semver
    |   `-- xml-mapping
    |-- testData
    |   |-- deviceReporter
    |   |-- ontologies
    |   |-- preferences
    |   |-- security
    |   `-- solutions
    `-- tests
        |-- configs
        |-- data
        |-- platform
        |-- shared
        `-- web